server/src/fat/java/com/ibm/ws/lars/rest/ApiTest.java

```diff
@@ -1146,11 +1209,16 @@ public void test500Mapping() throws InvalidJsonAssetException, ParseException, I
     @Test
     public void testAssetReviews() throws InvalidJsonAssetException, IOException {
         Asset a = addLittleAsset("weather", "hot", "ground", "flat", "description", "lovely");
-        List<?> ars = repository.getAssetReviews(a.get_id());
+        List<Map<String, Object>> ars = repository.getAssetReviews(a.get_id());
         // Story 165844, for now assetreviews just returns an empty JSON array
-        assertEquals(
-                     "Should be an empty List",
-                     0, ars.size());
+        assertEquals("Should be an empty List", 0, ars.size());
+        RepositoryContext userRepo = RepositoryContext.toUserContext(repository);
+        try {
```

Comment:
Generally, rather than catching the AssertionError, we've had two methods for each operation on RepositoryContext: one that expects to succeed and returns the result and another that expects to fail with a given error code and returns nothing.